dist/5655-schema.json

```diff
@@ -261,24 +261,68 @@
         "string",
         "null"
       ]
+    },
+    "ssn": {
+      "type": "string",
+      "pattern": "^[0-9]{9}$"
+    },
+    "employmentHistory": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "veteranORSpouse": {
+            "type": "string"
+          },
+          "occupationName": {
+            "type": "string"
+          },
+          "from": {
+            "type": "string"
```

Comment:
Is `date` an option here?

Reply:
Not at the moment as these dates will consist only of month and year and there is still some ongoing discussion about how those will be represented. I will write a regex to capture the correct format once it is decided.